AutomationPracticeMainPage.java

```diff
@@ -0,0 +1,45 @@
+package pageobject.AutomationPracticeSite;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import pageobject.AbstractPage;
+
+public class AutomationPracticeMainPage extends AbstractPage {
+
+    @FindBy(xpath = "(//div[@class='product-container'])[7]")
+    private WebElement productItem;
+
+    @FindBy(xpath = "(//a[@class='product-name'][@title='Printed Chiffon Dress'])[1]")
+    private WebElement productTitle;
+
+    @FindBy(xpath = "//a[@title='Printed Chiffon Dress']")
+    private WebElement productImageItem;
+
+    @FindBy(xpath = "(//span[@class='price product-price'])[14]")
+    private WebElement productPrice;
+
+    public AutomationPracticeMainPage(WebDriver webDriver) {
+        super(webDriver);
+    }
+
+    public boolean isPresentOfProductItem() {
+        webDriverWait.until(ExpectedConditions.visibilityOf(productItem));
+        boolean item = productItem.isDisplayed();
+        return item;
+    }
+
+    public String getProductPriceText() {
+        webDriverWait.until(ExpectedConditions.visibilityOf(productPrice));
+        String text = productPrice.getText();
```

Comment:
trim() could be done right after getting text, unless you want to make sure it is not null

Reply:
I agree, it would be the right decision.